js/narrativeView.js

```diff
@@ -185,7 +185,20 @@ define([
 
       if (this._isInitial) return;
 
-      const $elementToFocus = this.isLargeMode() ? this.$(`.narrative__content-item[data-index="${itemIndex}"]`) : this.$('.narrative__strapline-btn');
+      if ($straplineHeaderElm.css('transitionDuration') !== '0s') {
+        $straplineHeaderElm.one('transitionend', () => {
+          this.focusOnNarrativeElement(itemIndex);
+        });
+      } else {
+        this.focusOnNarrativeElement(itemIndex);
+      }
```

Comment:
```suggestion
      const hasStraplineTransition = !this.isLargeMode() && parseFloat($straplineHeaderElm.css('transitionDuration'));
      if (hasStraplineTransition) {
        $straplineHeaderElm.one('transitionend', () => {
          this.focusOnNarrativeElement(itemIndex);
        });
      } else {
        this.focusOnNarrativeElement(itemIndex);
      }
```
Without the isLarge check in large mode it'll always try to wait for the strapline to finish animating, but it never will, so we're just queuing up lots of once event handlers which never callback.
parseFloat is just to check there is any numeric value larger than 0, regardless of unit.